README.md

```diff
@@ -6,4 +6,31 @@
 * `python3 -m venv venv`
 * `source venv/bin/activate`
 * `pip3 install -r requirements.txt`
-* `python3 app.py`
\ No newline at end of file
+* `python3 app.py`
+
+
+
+## Docker Infrastructure:
+The docker compose config in this repository is intended to provide a small/simple suite of services for TunnelVision to rely on. This is for development and testing purposes.
+
+To use this suite:
+
+1. create a file called `compose.env` in the root of the repository. Use the following template to get started:
+
+```
+MINIO_ROOT_USER=
+MINIO_ROOT_PASSWORD=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+```
+2. fill in appropriate values
+3. `docker compose up`
+4. navigate to http://localhost:9001, log in with the root credentials for minio specified above, add create a bucket for TunnelVision
+5. while still in the minio console, navigate to "access keys" on the left and create an access key and secret for tunnelvision to use.
```

Comment:
This doesn't include any steps for setting up the database. I'm not sure if there's a way to auto-create it from the `docker-compose` file, but currently it need to be done manually using `createdb -h localhost -U <postgres user> tunnelvision`. Additionally, the actual database tables need to be created as well. Setting up `flask-migrate` might be able to handle that.

Reply:
I believe using flask is how i did it - because this uses SQLAlchemy, the `db.create_all()` call will create the tables.

i also have a separate branch that gets flask-migrate working for versioning/handling changes to the schema, since the method i just described only goes from an empty db to the current schema